### ISSUE TITLE
calibratePeltier.cpp does not compile

### ISSUE THREAD

**Issue by dmiklic** [larics/assisi-playground#57]
The calibratePeltier.cpp that's currently in the master branch does not compile. The error is:

```
/home/damjan/devel/assisi/playground/playground/calibratePeltier.cpp:18:40: error: no matching function for call to ‘Enki::Casu::Casu(Enki::WorldExt*&, double&)’
  Casu *casu = new Casu (world, env_temp);
                                        ^
/home/damjan/devel/assisi/playground/playground/calibratePeltier.cpp:18:40: note: candidates are:
In file included from /home/damjan/devel/assisi/playground/playground/calibratePeltier.cpp:5:0:
/home/damjan/devel/assisi/playground/robots/Casu.h:79:9: note: Enki::Casu::Casu(Enki::Vector, double, Enki::ExtendedWorld*, double, int)
         Casu (Vector pos, double yaw, ExtendedWorld* world, double ambientTemperature, int bridgeMask = 0);
         ^
/home/damjan/devel/assisi/playground/robots/Casu.h:79:9: note:   candidate expects 5 arguments, 2 provided
/home/damjan/devel/assisi/playground/robots/Casu.h:28:11: note: Enki::Casu::Casu(const Enki::Casu&)
     class Casu : public ExtendedRobot
           ^
/home/damjan/devel/assisi/playground/robots/Casu.h:28:11: note:   candidate expects 1 argument, 2 provided
```

@plsm Please fix.

**Comment by rmm-fcul** [comment]
As a temporary workaround it is possible to get the playground to compile with:
    make assisi_playground
rather than the implicit `make all`

**Comment by plsm** [comment]
I have removed the calibratePeltier program. I have created a pull request for this.

**Comment by dmiklic** [comment]
I guess this got lost somewhere among all the pull requests of last week :) @plsm please sync with the master branch, fix this and make another pull request.

**Comment by dmiklic** [comment]
@plsm can you please take care of this?

**Comment by plsm** [comment]
I have created a pull request.

**Comment by dmiklic** [comment]
Closed by  13bf673
